src/zenml/integrations/pytorch/materializers/pytorch_materializer.py

```diff
@@ -36,7 +36,7 @@ def handle_input(self, data_type: Type[Any]) -> Union[Module, TorchDict]:
             A loaded pytorch model.
         """
         super().handle_input(data_type)
-        return torch.load(os.path.join(self.artifact.uri, DEFAULT_FILENAME))  # type: ignore[no-untyped-call] # noqa
+        return torch.load(os.path.join(self.artifact.uri, DEFAULT_FILENAME))
```

Comment:
mypy complains because this ignore comment was removed

Reply:
Fixed in the latest commits